plugins/@grouparoo/clickhouse/__tests__/table-import/import-properties.ts

```diff
@@ -362,6 +362,7 @@ describe("clickhouse/table/recordProperties", () => {
           aggregationMethod: AggregationMethod.Exact,
         });
         expect(values[record.id][properties[0].id]).toEqual([
+          null,
```

Comment:
Shenanigans going on with the order of the responses when one of the rows contains a`null` in this Clickhouse test... is there a looser way I can match the array (IE: by contents but not necessarily order?)

Reply:
We do this in some of other tests with a `sort()` -> `expect(name.stort()).toEqual(['a','b'].sort())`